session/session_linux.go

```diff
@@ -96,8 +96,8 @@ func (s *Session) Namespace() (error, error) {
 	if len(s.mopts) > 0 {
 		opts += "," + s.mopts
 	}
-	v("CPUD: mount 127.0.0.1 on /tmp/cpu 9p %#x %s", flags, opts)
-	if err := unix.Mount("localhost", "/tmp/cpu", "9p", flags, opts); err != nil {
+	v("CPUD: mount 127.0.0.1 on %s/cpu 9p %#x %s", s.tmpMnt, flags, opts)
+	if err := unix.Mount("localhost", s.tmpMnt+"/cpu", "9p", flags, opts); err != nil {
```

Comment:
pretty much everywhere you've to any kind of assembling of file names, use filepath.Join

There's a very small chance cpud could run on non-linux systems some day.

Reply:
Done.